features/375-static-cdn/config.md

```diff
@@ -10,6 +10,7 @@ deployments:
     path: /
     context: ./frontend/build
     error_page: /404.html
+    index_file: index.htm
```

Comment:
Not sure if [error_page](http://nginx.org/en/docs/http/ngx_http_core_module.html#error_page) is following nginx's convention. If yes, we should also follow nginx to call this [index](http://nginx.org/en/docs/http/ngx_http_index_module.html#index) instead of `index_file`

Reply:
Not intentionally, and since providers have different name for this configuration (e.g. ErrorDocument for S3/Apache, error_page for nginx), I don't think it is necessary to follow any one of them.

Reply:
ummm, I do want to clarify (or maybe we need it on the doc), does `error_page` and `index_file` accept same type of parameters, reading it feel like "page" meant a URL (and the example is `/404.html`, while `file` is a file name... is it intended? Maybe need to clarify it and consider it with the naming too...

Reply:
Yes, they accept different value type. `error_page` is an absolute path to the error page, and `index_file` is the file name of the index page.